assets/scss/common/_dark.scss

```diff
@@ -183,6 +183,7 @@ $navbar-dark-active-color:          $link-color-dark;
   color: $body-color-dark;
   background-color: transparent;
   border: 0;
+  text-transform: capitalize;
```

Comment:
The _dark.scss file only applies to dark mode, please double check your work in light mode.

Reply:
Thanks! Will check it now itself!